### ISSUE TITLE
[UI/OnnxSamples] The AIs dont work on WASM

### ISSUE THREAD

**Issue by Marc-Antoine-Soucy** [unoplatform/Uno.Samples#481]
![image](https://github.com/unoplatform/Uno.Samples/assets/90481654/8d502df0-7eaa-4ac5-a837-20d71c3b75b9)

**Comment by Marc-Antoine-Soucy** [comment]
If this is not fixable (and I don't think it is) I guess we should not have a WASM project for OnnxSamples?

**Comment by nickrandolph** [comment]
That's correct, don't have a WASM project if we can't get this to work on web. Can you investigate whether it's possible to run onnx in wasm?
